tests/ee_acrablock_config.yaml

```diff
@@ -16,7 +16,7 @@ schemas:
       - masked_prefix
     encrypted:
       - column: specified_client_id
-        client_id: keypair1
+        client_id: 12824c1c541a615f428a740770291374a8504f84a0682ab4015bc8e792b1bc8240022c5c9faa8c2111e0076b3b929148f4b801414413edaa800cb42492c20cf7
```

Comment:
please, add comments that it's hash of tls certificate with relative path to it. it will simplify understanding how to reproduce or what we should use to re-generate if certs will be changed.

Reply:
Sure, do you want to add comments in all corresponded  config files?

Reply:
don't forget to add a comment. because now it's omitted for all of files.

Reply:
Sure, thanks for reminder